h2o-algos/src/main/java/water/rapids/RapidsInit.java

```diff
@@ -0,0 +1,11 @@
+package water.rapids;
+
+import water.rapids.prims.word2vec.AstWord2VecToFrame;
+
+public class RapidsInit {
+
+  public static void registerAlgoRapids() {
+    Env.init(new AstWord2VecToFrame());
```

Comment:
Thinking about it more: 

If we go through SPI way, we need some interface/abstract class here, and something like EnvContext, which would permit registration:

```java
public class W2VRapidsInit implements RegisterRapids {
  @Override public  void registerAlgoRapids(EnvContext envContext) {
    envContext.init(new AstWord2VecToFrame());
  }
}
```

where EnvContext is just dummy class:
```java
class DummyEnvCOntext implements EnvContext {
  @Override public void init(AstOp astOp) {
     Env.init(astOp); // static call
  }
}```

WDYT?